cypress/integration/BounceRulesDetailed/bounce_rules_detailed.spec.js

```diff
@@ -0,0 +1,45 @@
+import BounceRuleDetailed from "../../Pages/BounceRulesDetailed";
+
+const testCreateRule = {
+  priority: 2,
+  bounce_action: "cypressDetailedTest",
+  response_code: 281,
+  description: "cypressDetailedTest",
+  enhanced_code: "384",
+  regex: "kjfgsdlfg",
+};
+let ruleId;
+
+describe("Bounce Rule Detailed", () => {
+  before(async () => {
+    ruleId = await BounceRuleDetailed.createTestRuleAPI(testCreateRule);
+  });
+
+  beforeEach(() => {
+    BounceRuleDetailed.open(ruleId);
+  });
+
+  after(() => {
+    BounceRuleDetailed.deleteBounceRuleAPI(ruleId);
+  });
+
+  it("should pass healthchecks", () => {
+    BounceRuleDetailed.details.should("be.visible");
+    BounceRuleDetailed.changelog.should("be.visible");
+    BounceRuleDetailed.editButton.should("be.visible");
+  });
+
+  it("should edit a bounce rule", () => {
+    BounceRuleDetailed.updateRule().then(() => {
```

Comment:
So after updating the updateRule function to take in an object with all the fields to fill out it'll look a lot better and won't require you to know what it's hardcoded to in the spec like
```js
const updatedRule = {
  description: "...",
  commit: "...",
};
BounceRuleDetailed.updateRule(updatedRule).then(() => {
BounceRuleDetailed.testChangelog.should("contain", updatedRule.description);
BounceRuleDetailed.testChangelog.should("contain", updatedRule.commit);
})
```